issue_1/contracts/WordRegistry.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4;
+
+import "hardhat/console.sol";
+
+import "./yield-utils-v2/contracts/access/Ownable.sol";
+
+/**
+@title Dictatorship-friendly word registry
+@author Owleksiy
+@notice This contract allows users to register/unregister (claim/unclaim) words
+Once a word is regiistered by a user, other users can't register it, unless the
+original user unregisters it.
+
+Additionally, the owner of the contract can expropriate any registered word
+ */
+contract WordRegistry is Ownable {
+    event OwnershipChange(string word, address previousOwner, address newOwner);
```

Comment:
It's easy to misunderstand this event with `OwnershipTransferred`  from `Ownable`. Maybe use something like `NameRegistered`?

Reply:
ack